packages/vuepress-theme-nimatron/styles/index.scss

```diff
@@ -257,3 +257,39 @@ h1 {
     margin: 0;
   }
 }
+
+.content__default { // stylelint-disable
+  > h1,
+  > h2,
+  > h3,
+  > h4,
+  > h5,
+  > h6 {
+    position: relative;
+
+    .header-anchor {
+      position: absolute;
+      left: -30px;
+      width: 30px;
+      opacity: 0;
+      text-align: center;
+      text-decoration: none;
+    }
+
+    &:hover {
+      .header-anchor {
+        opacity: 1;
+      }
+    }
+  }
+}
+
+.fade-leave-active,
+.fade-enter-active {
+  transition: opacity 0.5s;
```

Comment:
Can we use our default animation variables here? Not important, but easier for updates.

Reply:
We can, im actually going to remove the animation from the theme. Was just a thing that I did to see how vue handles route transitions.